flight/Modules/GPS/GPS.c

```diff
@@ -234,6 +230,9 @@ static void gpsTask(void *parameters)
 	}
 #endif /* PIOS_GPS_MINIMAL */
 
+	timeOfLastUpdateMs = 0;
+	timeOfLastCommandMs = PIOS_Thread_Systime();
```

Comment:
I added a goto... so we can go back up above this point and update the time to the last initialization.